aspnetcore/performance/caching/response.md

```diff
@@ -1,92 +1,72 @@
-
-![image](response/_static/proxy-and-cache.png)
-
-The primary HTTP header used for caching is `Cache-Control`. The [HTTP 1.1 specification](https://tools.ietf.org/html/rfc7234#section-5.2) details many options for this directive. Three common directives are:
-
-**public**
-
-   Indicates that the response may be cached.
+*Response caching* adds cache-related headers to responses. These headers specify how you want client, intermediate (proxy) machines and middleware to cache responses. Response caching can reduce the number of requests a client or proxy makes to the web server. Response caching can also reduce the amount of work the web server performs to generate the response. Repeated request URLs that match may be served from the cache of the client, proxy or server. Cached responses can be returned by the client, the proxy or served by the response caching middleware. Cached requests served by the client or proxy save bandwidth and server load. Cached requests served by middleware can reduce web app server load.
 
-**private**
+The primary HTTP header used for caching is `Cache-Control`. See the [HTTP 1.1 Caching](https://tools.ietf.org/html/rfc7234#section-5.2) and [Cache-Control](https://www.w3.org/Protocols/rfc2616/rfc2616-sec14.html#sec14.9) for more information. Common cache directives:
 
-   Indicates the response is intended for a single user and **must not** be cached by a shared cache. The response could still be cached in a private cache (for instance, by the user's browser).
-
-**no-cache**
-
-   Indicates the response **must not** be used by a cache to satisfy any subsequent request (without successful revalidation with the origin server).
-
-> [!NOTE]
-> **Response caching does not cache responses on the web server**. It differs from [output caching](http://www.asp.net/mvc/overview/older-versions-1/controllers-and-routing/improving-performance-with-output-caching-cs), which would cache responses in memory on the server in earlier versions of ASP.NET and ASP.NET MVC. Output caching middleware is planned to be added to ASP.NET Core in a future release.
+* [public](https://tools.ietf.org/html/rfc7234#section-5.2.2.5)
+* [private](https://tools.ietf.org/html/rfc7234#section-5.2.2.6)
+* [no-cache](https://tools.ietf.org/html/rfc7234#section-5.2.1.4)
+* [Pragma](https://tools.ietf.org/html/rfc7234#section-5.4)
+* [Vary](https://tools.ietf.org/html/rfc7231#section-7.1.4)
 
-Additional HTTP headers used for caching include `Pragma` and `Vary`, which are described below. Learn more about [Caching in HTTP from the specification](https://tools.ietf.org/html/rfc7234#section-3).
+Caching responses on the web server can be enabled by adding the [response caching middleware](https://github.com/aspnet/responsecaching). See [Response caching middleware](middleware.md) for more information.
```

Comment:
I don't think we usually link to the repo right? I had that in the original PR since I didn't have a link to any docs regarding it. Now that we at least have a place holder, I think that is sufficient.